src/movies.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from 'react'
+import Button from 'react-bootstrap/Button';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import axios from 'axios'
+
+export default class Movie extends Component {
```

Comment:
Feedback here is about task card for lab 9. It asks us to:

- Create a nested `Movie` component (inside `Movie` component) for rendering of each individual movie object found in the raw response data.